### PR TITLE
Swclfw 12

### PR DESCRIPTION
**Pull request by DanBurkhardt**
separated all types from URLSession.swift into interceptablesession etc., fixed all privacy / ACL errors (updated fileprivate to internal), removed test example file from initial project setup, fixed all but two annoying compiler warnings

### CODE REVIEW

File: Source/SwiftCloudant/HTTP/InterceptableSession/InterceptableSessionConfiguration.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
+//
+//	File.swift
```
Comment (emilydoran1) [inline Source/SwiftCloudant/HTTP/InterceptableSession/InterceptableSessionConfiguration.swift]: Forgot to update 'File.swift'

**Review by emilydoran1 — COMMENTED** [review]
Other than the 1 file which needs an updated header, looks great!

File: Source/SwiftCloudant/HTTP/InterceptableSession/InterceptableSessionConfiguration.swift (hunk 1 above)
Comment (DanBurkhardt) [reply Source/SwiftCloudant/HTTP/InterceptableSession/InterceptableSessionConfiguration.swift]: ahhhh, thanks for the catch 😆. will fix.

**Review by emilydoran1 — APPROVED** [review]
Looks good, thanks!

**Pull request merged**